pages/Screenshots.html

```diff
@@ -19,29 +19,29 @@ <h3>
     </a>
     Main page
   </h3>
-  <p><img src="static/images/main.png" alt="Screenshot of main page" class="screenshot-image"></p>
+  <p><img src="static/images/main.png" alt="Main page having two green buttons reading, 'Create your own lesson' and 'Browse our lessons' respectively. The top-navigation bar consists of the Oppia logo on the left hand corner and buttons for 'Library', 'About', 'Donate', 'Create' and 'Sign-in' on the right hand side." class="screenshot-image"></p>
 
   <h3>
     <a id="gallery" class="anchor" href="#gallery" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
     Library
   </h3>
-  <p><img src="static/images/library.png" alt="Screenshot of library page" class="screenshot-image"></p>
+  <p><img src="static/images/library.png" alt="A banner reading 'Imagine what you could learn today...', followed by a panel showcasing featured activities." class="screenshot-image"></p>
 
   <h3>
     <a id="learner-view" class="anchor" href="#learner-view" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
     Learner view
   </h3>
-  <p><img src="static/images/exploration.png" alt="Screenshot of learner view" class="screenshot-image"></p>
+  <p><img src="static/images/exploration.png" alt="Two white cards placed adjacently, one containing a blue box with a descriptive method to play the first note, as part of the music theory exploration. The other having a white 'play' button on the top left corner, followed by a single musical note." class="screenshot-image"></p>
```

Comment:
Don't understand what "descriptive method to play the first note" means, can you explain using clearer, simpler language?

"The other card has a white 'play' button on the top left corner, followed by three musical notes on a staff."